src/api/routers/participantsRouter.ts

```diff
@@ -272,6 +280,55 @@ export function createParticipantsRouter() {
     }
   );
 
+  participantsRouter.get(
+    '/:participantId/apiRoles',
+    async (req: ParticipantRequest, res: Response) => {
+      const { participant } = req;
+      if (!participant?.siteId) {
```

Comment:
Is this check strictly necessary?

Reply:
Participants is not guaranteed and it having a site is not a guaranteed so I believe it is good to have.

Reply:
yes, while a participant should have a `siteId`, I don't think we need to check it here. I think this endpoint should still succeed even if the participant does not have a `siteId`. Other endpoints in this file rely on a `siteId` because they are making calls to the admin service, but this one is just calling our db. Does that make sense?

Reply:
Ah yeah that makes sense.

Reply:
Wait sorry I just had another look and it does need it. Since it's calling admin to create the key. See line 322.